implementations/cpp-jsontoolkit/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM alpine AS builder
```

We pin these base images so that the builds are reproducible, and then configure dependabot to update them (which is what pre-commit is failing to say).

So

```suggestion
FROM alpine:3.20 AS builder
```

(and below).

---

Ah, I copied it from `valijson`, which seems to work without warnings. I'll pin it!